LeanKit-Labs/tag-release
# Pre-releases! Much wow! Very Amazed!

Added the ability to do pre-releases with tag-release! This is a useful feature for developers, or should be so they can easily create tags for branches of work that need to be deployed and tested quickly.

Whoever reviews this PR please include @RobertReaves during the review as he has several test cases to describe that aren't trivial to describe.

Testing:

1) Link this PR's version of tag-release to your own local versoin.
2) Create a local and upstream feature-branch that is based off of master for a throw-away repo.
3) Make changes to your local branch and create a commit.
4) $ tag-release -p ( from your feature-branch )
5) Follow the questions and use the hints provided to make sure it is working as you expect.
6) Once finished you should now have a pre-release tagged version of your code! Yay!
